frontend/src/tabs/authorship.pug

```diff
@@ -57,17 +63,23 @@
       .file.active(v-bind:key="file.path")
         .title
           span.path(onclick="toggleNext(this.parentNode)", v-on:click="updateCount") {{ file.path }}&nbsp;
-          span.loc ({{ file.lineCount }} lines)
-          a(
-            v-bind:href="getFileLink(file, 'commits')", target="_blank",
-            title="click to view the history view of file"
-          )
-            i.button.fas.fa-history
-          a(
-            v-bind:href="getFileLink(file, 'blame')", target="_blank",
-            title="click to view the blame view of file"
-          )
-            i.button.fas.fa-user-edit
+          span.icons
+            a(
+              v-bind:href="getFileLink(file, 'commits')", target="_blank",
+              title="click to view the history view of file"
+            )
+              i.button.fas.fa-history
+            a(
+              v-bind:href="getFileLink(file, 'blame')", target="_blank",
+              title="click to view the blame view of file"
+            )
+              i.button.fas.fa-user-edit
+          span.fileTypeLabel(
+            v-bind:style="{\
+              'background-color': fileTypeColors[file.fileType],\
+              'color': this.getFontColor(fileTypeColors[file.fileType])\
+              }"
```

Comment:
the indentation of closing braces should be in the same line as `v-bind`?

Reply:
If you consider the closing braces just as a part of the string, then it should be 2 space after v-bind? Following the style in ramp.pug line 17

Reply:
hmmmm right, i was referring to line 42 of ramp.pug. Nvm then, ignore the comments